openapi/openapi.yaml

```
@@ -1241,6 +1241,10 @@ paths:
     $ref: paths/customers@{id}@edd-timeline.yaml
   '/customers/{id}/edd-search-results':
     $ref: paths/customers@{id}@edd-search-results.yaml
+  '/tags-configurations':
```

```suggestion
  '/tags-rules':
```
For example this is what I thought should be renamed..

---

Done, thank you.